File: easy/hovercraft/hovercraft.rb
"""
@author	André Peralta
@version 20/05/2021
"""

def profits sales 

	insurance = -1000000
	profit = 3000000 * sales
	cost = -10*2000000

	final_profit = insurance+cost+profit

	if final_profit > 0
		return "Profit"
	elsif final_profit < 0
		return "Loss"
	elsif final_profit == 0
		return "Broke Even"
		
	end
end


puts "Sales: "
sales = gets.to_i

print profits(sales)

File: easy/hovercraft/hovercraft.c
#include <stdio.h>

int main(){

    int initialCost = 2000000*10;
    int sales;

    printf("Número de aerodeslizadores: ");
    scanf("%d",&sales);

    sales *= 3000000;
    sales -= 1000000;

    if(sales > initialCost) printf("Profit");
    else if(sales < initialCost) printf("Loss");
    else if(sales == initialCost) printf("Broke Even");
    return 0;
}
